serverless/src/getKeyword/handler.js

```diff
@@ -0,0 +1,58 @@
+import { createPrefLabelMap } from '@/shared/createPrefLabelMap'
+import { getApplicationConfig } from '@/shared/getConfig'
+import { getSkosConcept } from '@/shared/getSkosConcept'
+import { toKeywordJson } from '@/shared/toKeywordJson'
+
+const getKeyword = async (event) => {
+  // Extract configuration and parameters
+  const { defaultResponseHeaders } = getApplicationConfig()
+  const queryStringParameters = event.queryStringParameters || {}
+  const version = queryStringParameters?.version || 'published'
+  const { conceptId } = event.pathParameters
+
+  try {
+    const concept = await getSkosConcept({
+      conceptIRI: `https://gcmd.earthdata.nasa.gov/kms/concept/${conceptId}`,
+      version
+    })
+
+    // Check if concept is null and return 404 if so
+    if (concept === null) {
+      return {
+        statusCode: 404,
+        headers: {
+          ...defaultResponseHeaders,
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          error: 'Keyword not found'
```

Comment:
could we put the actual keyword in the error message so we can debug from Splunk (if that is where this would be seen)

Reply:
Good idea, now added. Still need to add those to splunk log for all handler.